src/applications/vaos/containers/DateTimeRequestPage.jsx

```diff
@@ -42,19 +47,18 @@ export class DateTimeRequestPage extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
+    const { validationError } = this.state;
     if (
-      this.state.validationError &&
-      !prevState.submitted &&
-      this.state.submitted
+      validationError &&
+      (this.isMaxSelectionsError() ||
+        (!prevState.submitted && this.state.submitted))
     ) {
       scrollAndFocus('.usa-input-error-message');
```

Comment:
I'll add my screenshot here. Let's remove the yellow focus halo from the `<span class="usa-input-error-message">` because it's not a typical focusable element. I'd also like to add `role="alert"` to that span, so it announces the error message assertively to all screen readers. VoiceOver missed it once because it was still reading out the input instructions.

---
![Screen Shot 2020-03-06 at 10 22 45 AM](https://user-images.githubusercontent.com/934879/76104114-6442f980-5f98-11ea-8512-cb924f136359.png)